### ISSUE TITLE
Stuck With 1 Worker

### ISSUE THREAD

**Issue by mat-adamec** [CoffeaTeam/coffea-casa#78]
I'm using the default sidecar scheduler which ranges from 1 to 100 workers, and I haven't turned autoscaling off, but I only ever seem to get 1 worker when I run anything. I'll update here if this persists or fixes itself, but wanted to point it out so that we can better pinpoint when the issue started if it does persist.

**Comment by mat-adamec** [comment]
I tried manually requesting more than one worker with:

```
from coffea_casa import CoffeaCasaCluster
cluster = CoffeaCasaCluster()
cluster.scale(jobs=10) 
from dask.distributed import Client
client = Client(cluster)
```

But still only received one, I believe. In the processing log, there was only:

distributed.core - INFO - Starting established connection

[                                        ] | 0% Completed |  8.1s

distributed.comm.tcp - INFO - Connection closed before handshake completed

once, and I seem to recall it printing for every worker when you set it up manually. It also took forever - so I doubt I actually got 10.
